bcda/web/router_test.go

```diff
@@ -76,20 +75,6 @@ func (s *RouterTestSuite) TestDataRoute() {
 	assert.Equal(s.T(), http.StatusUnauthorized, res.StatusCode)
 }
 
-func (s *RouterTestSuite) TestFileServerRoute() {
```

Comment:
Test replaced by: https://github.com/CMSgov/bcda-app/pull/606/files#diff-842f2e0111020a6f5af8ab0ad66d148de552df214ba184469f9587155f9d5337R175

Reply:
I can't get this link to work for me.  It takes me back to the top of this pull request.

Reply:
Thanks for the help guiding to the right spot.  LGTM.